spec/features/suspected_plagiarism_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe 'Suspected Plagiarism', type: :feature, js: true do
+  let(:course) { create(:course, start: '2017-01-01', end: '2020-01-01') }
+  let(:user) { create(:user, username: 'Ragesoss') }
+  let(:article) { create(:article, title: 'Nancy_Tuana') }
+
+  before do
+    course.campaigns << Campaign.first
+    create(:courses_user, course: course, user: user)
+    create(:revision, article: article, user: user, date: '2019-01-01')
+    create(:articles_course, course: course, article: article, user_ids: [user.id])
+  end
+
+  it 'should show an empty list when there are no possible plagiarism' do
+    visit "/courses/#{course.slug}/activity/plagiarism"
+    within(:css, '.possible_plagiarism', wait: 20) do
```

Comment:
Looking for specific elements based on their css and xpath selectors is okay, but when it's possible to do without them, it makes for a spec that is more readable, more focused on a user perspective, and less likely to break when the related interface components get changed.

In this case, instead of looking for a table with a single entry, I suggest looking for the unique text of the 'no suspected plagiarism' message.